src/openeo_aggregator/backend.py

```diff
@@ -125,52 +151,95 @@ def _merge_collection_metadata(self, by_backend: Dict[str, dict]) -> dict:
 
         result = {
             "id": cid,
+            "stac_version": max(list(getter.get("stac_version")) + ["0.9.0"]),
+            "title": getter.first("title", default=cid),
+            "description": getter.first("description", default=cid),
+            "type": getter.first("type", default="Collection"),
+            "links": [l for l in list(getter.concat("links")) if l.get("rel") not in ("self", "parent", "root")],
+            "summaries": getter.select("summaries").simple_merge()
```

Comment:
doing general "simple" merge on all summaries is bit dangerous I think: it depends on summary type: e.g. for eo:bands, doing concat is bad idea, for others, just taking first is good enough

Reply:
eliminated simple_merge and moved appropriate logic to each of the summary fields